.github/workflows/main.yml

```diff
@@ -68,6 +68,27 @@ jobs:
           GATSBY_EXPERIMENTAL_QUERY_CONCURRENCY: 8 # default is 4, going too high on the 2-core github actions machines will fire back
           GATSBY_CPU_COUNT: logical_cores # use hyperthreading if available to statistically better saturate the cores
 
+      - name: Deploy to Vercel (Preview)
```

Comment:
To ease the dev workflow (was relevant in this branch): deploy before linting to have a preview even if there are stylistic code issues.